services/core-api/app/config.py

```diff
@@ -25,8 +25,8 @@ def get_key_cloak_client_id():
                     if getJwtManager().audience:
                         return getJwtManager().audience
             except Exception as e:
-                # Handle the exception here (e.g., log it)
-                print(f"An error occurred: {e}")
+                # The exception is caught, but no error message is printed or logged
```

Comment:
I did find that I was able to log both the exception message and the stack trace here in the online editor.

But logging the exception message was just a tiny bit tricky. You can access it with `e.error` _but_ if the same method catches something like ValueError('text'), it'll have extra errors because .error doesn't exist, while directly logging the error does work in that case.

The stack trace was nice and simple!
Just import traceback and put in `print(traceback.format_exc())`.